kernel/kernel-api/src/main/java/io/delta/kernel/internal/tablefeatures/TableFeatures.java

```diff
@@ -262,8 +257,8 @@ public boolean metadataRequiresFeatureToBeEnabled(Protocol protocol, Metadata me
     }
 
     @Override
-    public boolean hasKernelWriteSupport(Metadata metadata) {
-      return false; // TODO: yet to be implemented in Kernel
+    public Set<TableFeature> requiredFeatures() {
```

Comment:
Should we test that we enable dependent features here? Not sure where we test this for other table features

Reply:
Sure, I updated ProtocolSuite which looks like it was doing some tests around dependent table features for a given feature.